### PR TITLE
fixed missing references

### PR DESCRIPTION
**Pull request by the-vampiire**
- trimmed the fat but forgot to trim the clothes...

need to merge into master for this one @TheSabby

### CODE REVIEW

File: slack/slackTools/slackApi/convertUserID.js [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
-      request.post({ url }, (error, response, responseBody) => {
-        const body = JSON.parse(responseBody);
-        if (error) reject(error);
-        else if (body.ok) {
-          resolve(body.user.profile.display_name);
-        } else reject('unexpected error');
-      });
-    }).catch(oAuthDocError => reject(oAuthDocError));
+    // use wizard to get oauth token
+    const oAuthToken = 'number_one_boolsheet';
```
Comment (TheSabby) [inline slack/slackTools/slackApi/convertUserID.js]: Are you sure the value of `oAuthToken` should be `number_one_boolsheet`?

File: slack/slackTools/slackApi/getChannelID.js [hunk 2]
```diff
@@ -1,23 +1,18 @@
...
-    oAuthModel.findOne({ teamID })
-    .then((result) => {
-      const oAuthToken = result.teamOAuthToken;
-      const url = `https://slack.com/api/channels.list?token=${oAuthToken}`;
-      request.post({ url }, (error, response, responseBody) => {
-        const body = JSON.parse(responseBody);
-        if (error) console.error(new Error(error));
-        else if (body.ok) {
-          const targetChannel = body.channels.find(slackChannel => slackChannel.name ===
+    const oAuthToken = 'numba one boolsheet';
```
Comment (TheSabby) [inline slack/slackTools/slackApi/getChannelID.js]: Here is a different version of it

File: slack/slackTools/slackApi/sendDialog.js [hunk 3]
```diff
@@ -13,25 +12,21 @@ const dialogBuilder = (type) => {
...
-      .then((oAuthDoc) => {
-        console.log(triggerID);
-        const dialogRequest = {
-          url: `https://slack.com/api/dialog.open?trigger_id=${triggerID}&dialog=${dialogBuilder(dialogType)}`,
-          headers: {
-            'Content-type': 'application/x-www-form-urlencoded',
-            Authorization: `Bearer ${oAuthDoc.teamOAuthToken}`,
-          },
-        };
+  const oAuthToken = 'boolsheet';
```
Comment (TheSabby) [inline slack/slackTools/slackApi/sendDialog.js]: 10/10

Comment (the-vampiire) [reply slack/slackTools/slackApi/sendDialog.js]: We can't all be number_one...

File: slack/slackTools/slackApi/convertUserID.js (hunk 1 above)
Comment (the-vampiire) [reply slack/slackTools/slackApi/convertUserID.js]: 💯💯

Comment (the-vampiire) [reply slack/slackTools/slackApi/convertUserID.js]: These are temps. Those methods will likely be replaced by the unholy filth of laze known as the SDK

**Pull request merged**